variables.tf

```diff
@@ -27,9 +27,14 @@ variable "node_count" {
   default     = 0
 }
 variable "instance_image" {
-  description = "The disk image to use when deploying new cloud instances"
+  description = "The disk image to use when deploying new cloud instances in the form of a full length Image ID or Marketplace URN"
   type        = string
-  default     = "CentOS-based 7.9 - Gen2"
+  default     = "almalinux:almalinux:8_5-gen2:latest"
```

Comment:
Could we use almalinux:almalinux:8-gen2:latest which by my understanding will take the latest image